boilerplate/main_service/services.py

```diff
@@ -31,12 +32,19 @@ def get_all_products():
 
     def get_product_info(product_id: int):
         """Method to get info of a product from the database"""
+        queryset = Product.objects.filter(id=product_id)
+        product_count = queryset.count()
+        if product_count == 0:
+            raise NotFoundError()
+        elif product_count > 1:
+            raise NotUniqueError()
+
+
         product = ProductService.get_single_product(product_id)
```

Comment:
I guess calling this method would also raise the `NotFoundError` and `NotUniqueError`? If so, maybe we don't need the codes from line 35-42?